app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,8 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find_by(id: params[:user_id])
+    @user = User.includes(:posts).find_by(id: params[:user_id])
     @posts = @user.posts
```

Comment:
- [ ] Greate work attempting to solve the **_N+1 queries_**. To remove the **_N+1 queries_** in the `index` action of the Posts controller, you could do this:

```ruby
 @posts = @User.posts.includes(:comments, :likes)
```

This will solve the N+1 query there.

--- 
Also, based on the requirements, you ought to attach screenshots of the terminal logs before fixing the **_N+1 queries_** and after fixing the **_N+1 queries_** in the pull request summary. Kindly implement these changes. 👍 😎